pyriemann/datasets/simulated.py

```diff
@@ -0,0 +1,111 @@
+import numpy as np
+from sklearn.utils.validation import check_random_state
+from .sampling import generate_random_spd_matrix, sample_gaussian_spd
+from pyriemann.utils.distance import distance_riemann
+from pyriemann.utils.base import powm, sqrtm
+
+
+def make_gaussian_blobs(n_samples=100, n_dim=2, class_sep=1.0, class_disp=1.0,
+                        random_state=None):
+    """Generate SPD dataset with two classes sampled from Riemannian Gaussian
+
+    Generate a dataset with SPD matrices generated from two Riemannian
```

Comment:
Repetition of word "generate":
```suggestion
    Generate a dataset with SPD matrices drawn from two Riemannian
```